Logger/FileLogger.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public FileLogger(string filePath)
+        {
+            FilePath = filePath;
+        }
+
+        public string FilePath { get; private set; }
```

Comment:
Prompt called for `FilePath` to be private.
```suggestion
        private string FilePath { get; set; }
```